src/partials.jl

```diff
@@ -82,6 +82,7 @@ Base.convert(::Type{Partials{N,V}}, partials::Partials{N,V}) where {N,V} = parti
 @inline Base.:-(a::Partials{N}, b::Partials{N}) where {N} = Partials(sub_tuples(a.values, b.values))
 @inline Base.:-(partials::Partials) = Partials(minus_tuple(partials.values))
 @inline Base.:*(x::Real, partials::Partials) = partials*x
```

Comment:
Can be deleted now?

Reply:
I think you're right.  @Keno can you confirm that you weren't leaving these specializations on `::Real` for some obscure reason?